### PR TITLE
[TD-11016]<docs>: add TDinsight in monitoring section of administrator page

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: documentation20/en/11.administrator/docs.md [hunk 1]
```diff
@@ -406,6 +406,51 @@ After TDengine is started, it will automatically create a monitoring database lo
 
 The collection of these monitoring metrics is turned on by default, but you can modify option monitor in the configuration file to turn it off or on.
 
+### TDinsight - Monitor TDengine with Grafana + Data Source
+
+Starting from v2.3.3.0, TDengine `log` database provides more metrics for resources and status monitoring. Here we are introducing a zero-dependency monitoring solution - we call it TDinsight - with only Grafana. The full documentations are located in [GitHub](https://github.com/taosdata/grafanaplugin/blob/master/dashboards/TDinsight.md).
```
Comment (sangshuduo) [inline documentation20/en/11.administrator/docs.md]: Suggestion:

Starting from v2.3.3.0, TDengine's `log` database provides more metrics for resources and status monitoring. Here we introduce a zero-dependency monitoring solution - we call it TDinsight - with Grafana. You can find the documentation from [GitHub](https://github.com/taosdata/grafanaplugin/blob/master/dashboards/TDinsight.md).

File: documentation20/en/11.administrator/docs.md [hunk 2]
```diff
@@ -406,6 +406,51 @@ After TDengine is started, it will automatically create a monitoring database lo
...
+
+For a shortcut, we provide a automation shell script [`TDinsight.sh`](https://github.com/taosdata/grafanaplugin/blob/master/dashboards/TDinsight.sh) to help setup TDinsight on Grafana server.
```
Comment (sangshuduo) [inline documentation20/en/11.administrator/docs.md]: Suggestion:
We provide an automation shell script [`TDinsight.sh`](https://github.com/taosdata/grafanaplugin/blob/master/dashboards/TDinsight.sh) as a shortcut to help setup TDinsight on the Grafana server.

File: documentation20/en/11.administrator/docs.md (hunk 1 above)
Comment (zitsen) [reply documentation20/en/11.administrator/docs.md]: Done

File: documentation20/en/11.administrator/docs.md (hunk 2 above)
Comment (zitsen) [reply documentation20/en/11.administrator/docs.md]: Done

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by boxiaobj — APPROVED** [review]
对社区版客户而言，挺好。
但对企业版客户而言，我们不能让客户去github上下载安装，应该考虑打包成为一个安装包。

**Pull request merged**